modin/engines/base/io/text/csv_glob_dispatcher.py

```diff
@@ -29,8 +34,30 @@
 
 
 class CSVGlobDispatcher(CSVDispatcher):
+    """Handles utils for reading multiple `.csv` files simultaneously. Inherits some common
+    for `.csv` files util functions from `CSVDispatcher` class.
+
+    """
+
     @classmethod
     def _read(cls, filepath_or_buffer, **kwargs):
+        """Read data from multiple `.csv` files passed with `filepath_or_buffer` simultaneously
+        according to the passed read_csv `kwargs` parameters. This function performs parameters
+        preprocessing, data file splitting, tasks launching and results postprocessing.
+
+        Parameters
+        ----------
+        filepath_or_buffer: str, path object or file-like object
```

Comment:
`filepath_or_buffer` also can be as list

Reply:
Not sure, if `filepath_or_buffer` is a list, `single_worker_read` function will be called, but `PandasCSVGlobParser.parse`, that is called underneath, doesn't expect list with filenames.

Reply:
oh, I see